### PR TITLE
Added delete.sh and replace.sh; Improved documentation

### LINKED ISSUE #5: Handle course renaming
Add and document the replacer script.

Copying from Trello:

> The solution to this problem is that we do the following:
> 
> 1. Take the site down
> 2. Get a database backup
> 3. Import the courses
> 4. Remove the old courses
> 5. Replace the courses IDs with the database backup
> 6. Import the database backups
> 
> This should make the course ID changes less problematic.

### PR DESCRIPTION
**Pull request by OmarIthawi**
Trello Card:

 - [Document and course removal and course ID replacement process](https://trello.com/c/hklHK065)

Closes #5

### CODE REVIEW

**Comment by OmarIthawi** [comment]
@appsembler/openedx FYI.

File: delete.sh [hunk 1]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+#!/usr/bin/env bash
```
Comment (thraxil) [inline delete.sh]: why the double shebang?

Comment (OmarIthawi) [reply delete.sh]: Thanks @thraxil for catching it. I've removed it!

**Review by tkeemon — APPROVED** [review]
This stuff is a real nightmare. 

The docs look great. My bash skills are a bit lacking, but the scripts look ok to me. I'd say let's go ahead and get this merged. The next time one of us needs to run through these steps, we can add/improve the docs as we're working through the process.

**Comment by OmarIthawi** [comment]
Thanks @tkeemon and @thraxil. Putting this in the master :truck:!

**Pull request merged**